app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+const {getAllTopics} = require('./controllers/topics_controller');
+const {
+    handleCustomErrors,
+    handlePsqlErrors,
+    handleServerErrors,
+  } = require('./errors/index.js');
+
+app.get('/api/topics', getAllTopics)
+
+app.all('/*', (request, response) => {
+    response.status(404).send({msg: "404 Not Found"});
+});
```

Comment:
Great way to handle the invalid endpoint. Would be even better if the error message is more specific like \endpoint does not exist' or 'path not found'